src/components/Tabs/Tabs.tsx

```diff
@@ -1 +1,53 @@
-export const Tabs = () => {};
+import React from 'react';
+import classNames from 'classnames';
+import { Tab } from '../../types/Tab';
+
+type Props = {
+  tabs: Tab[],
+  selectedTabId: string,
+  onTabSelected: (tabId: string) => void,
+  selectedTab: Tab,
+};
+
+export const Tabs: React.FC<Props> = ({
+  tabs,
+  selectedTabId,
+  onTabSelected,
+  selectedTab,
+}) => {
+  return (
```

Comment:
Use created `getSelectedTabById` function from helpers here to get the selected tab. In this case you'll be able to pass tests.

Reply:
My comments aren't fixed and your tests are failed.

Reply:
i thought that i didn't save, anyway, didn't understood correctly, but know i fix this problem, thanks, all tests passed:))